lib/schema/columncompiler.js

```diff
@@ -97,6 +97,26 @@ class ColumnCompiler {
     return modifiers.length > 0 ? ` ${modifiers.join(' ')}` : '';
   }
 
+  formatDefault(value) {
```

Comment:
What about lib/formatter/formatterUtils.js or somewhere around that?

Reply:
Sounds good!